### PR TITLE
poc for pip install -e

### PR DESCRIPTION
**Pull request by mhsieh**

### CODE REVIEW

**Comment by mhsieh** [comment]
@pb-cdunn this poc should work, once you specified pbfalcon_REPO in the settings, it will do:
1. `pip install -e`
2. workspace/pbfalcon would be a symbolic link to the REPO you specified.

File: ports/pacbio/pbfalcon/Makefile [hunk 1]
```diff
@@ -12,18 +12,16 @@ $(_WRKSRC):
 ifeq ($(wildcard $($(_NAME)_REPO)),)
 	$(GIT) clone $($(_NAME)_REPO) $@
 	cd $(_WRKSRC) && $(GIT) checkout $($(_NAME)_VERSION)
+DEVOPT =
 else
 	rm -f $(_WRKSRC)
 	ln -s $($(_NAME)_REPO) $(_WRKSRC)
+DEVOPT = -e
```
Comment (pb-cdunn) [inline ports/pacbio/pbfalcon/Makefile]: Slightly simpler and safer:
```sh
else
  ln -sf $($(_NAME)_REPO) $(_WRKSRC)
```

**Comment by pb-cdunn** [comment]
Works for me!

File: ports/pacbio/pbfalcon/Makefile (hunk 1 above)
Comment (mhsieh) [inline ports/pacbio/pbfalcon/Makefile]: okay will do.

**Comment by mhsieh** [comment]
```
mhsieh@login:~$ mkdir tmp
mhsieh@login:~$ cd tmp
mhsieh@login:~/tmp$ ls
mhsieh@login:~/tmp$ mkdir test
mhsieh@login:~/tmp$ ln -sf test test2
mhsieh@login:~/tmp$ ln -sf test test2
mhsieh@login:~/tmp$ ls -lG
total 4
drwxr-xr-x 2 mhsieh 4096 Apr 14 13:09 test
lrwxrwxrwx 1 mhsieh    4 Apr 14 13:09 test2 -> test
mhsieh@login:~/tmp$ ls -lG test/
total 0
lrwxrwxrwx 1 mhsieh 4 Apr 14 13:09 test -> test
```

**Comment by mhsieh** [comment]
I might be reverting this...

**Comment by pb-cdunn** [comment]
Revert? That's fine. No rush.

**Comment by mhsieh** [comment]
ln doesn't behave as I envisioned. See the test I did above.

**Comment by pb-cdunn** [comment]
Oh, yeah. `ln` has an ambiguous 2nd arg, which can we either a link or a directory into which to put the link. Add `-h`.

    ln -sfh dir link

That's available on OSX too.

**Comment by mhsieh** [comment]
ubuntu 1404's ln doesn't have -h, let me check...

**Comment by mhsieh** [comment]
bsd++

**Comment by mhsieh** [comment]
it was `ln -sfn`. I will do the update after this pbbam issue is fixed.

**Pull request merged**